light_curves/code_src/helpers/scale_up.py

```diff
@@ -283,79 +459,22 @@ def _now():
 
 
 def _load_yaml(yaml_file):
-    with open(yaml_file, "r") as fin:
-        yaml_dict = yaml.safe_load(fin)
-    return yaml_dict
-
-
-def write_kwargs_to_yaml(**kwargs_dict) -> None:
-    """Write `kwargs_dict` as a yaml file in the run's base_dir."""
-    yaml_path = run(build="yaml_file", **kwargs_dict)
-    with open(yaml_path, "w") as fout:
-        yaml.safe_dump(kwargs_dict, fout)
-    print(f"kwargs written to {yaml_path}")
-
-
-# ---- helpers for __name__ == "__main__" ---- #
-
-
-def _argparser():
```

Comment:
Did you mean to remove these?

Reply:
Looks accidental. Deleting these will break it.

Reply:
WOW!  so sorry, that is scary accidental.  I'm glad you caught that.

Reply:
no worries, this is why we do PR reviews :)